src/ReactiveUI/ObservableForProperty/OAPHCreationHelperMixin.cs

```diff
@@ -55,7 +54,50 @@ public static class OAPHCreationHelperMixin
             TRet initialValue = default(TRet),
             bool deferSubscription = false,
             IScheduler? scheduler = null)
-            where TObj : class, IReactiveObject => source.ObservableToProperty(target, property, initialValue, deferSubscription, scheduler);
+            where TObj : class, IReactiveObject
+            => ToProperty(target, source, property, () => initialValue, deferSubscription, scheduler);
+
+        /// <summary>
+        /// Converts an Observable to an ObservableAsPropertyHelper and
+        /// automatically provides the onChanged method to raise the property
+        /// changed notification.
+        /// </summary>
+        /// <typeparam name="TObj">The object type.</typeparam>
+        /// <typeparam name="TRet">The result type.</typeparam>
+        /// <param name="target">
+        /// The observable to convert to an ObservableAsPropertyHelper.
+        /// </param>
+        /// <param name="source">
+        /// The ReactiveObject that has the property.
+        /// </param>
+        /// <param name="property">
+        /// An Expression representing the property (i.e. <c>x => x.SomeProperty</c>).
+        /// </param>
+        /// <param name="getInitialValue">
+        /// The function used to retrieve the initial value of the property.
+        /// </param>
+        /// <param name="deferSubscription">
+        /// A value indicating whether the <see cref="ObservableAsPropertyHelper{T}"/>
+        /// should defer the subscription to the <paramref name="target"/> source
+        /// until the first call to <see cref="ObservableAsPropertyHelper{T}.Value"/>,
+        /// or if it should immediately subscribe to the <paramref name="target"/> source.
+        /// </param>
+        /// <param name="scheduler">
+        /// The scheduler that the notifications will be provided on - this should normally
+        /// be a Dispatcher-based scheduler.
+        /// </param>
+        /// <returns>
+        /// An initialized ObservableAsPropertyHelper; use this as the backing field
+        /// for your property.
+        /// </returns>
+        public static ObservableAsPropertyHelper<TRet> ToProperty<TObj, TRet>(
+            this IObservable<TRet> target,
+            TObj source,
+            Expression<Func<TObj, TRet>> property,
+            Func<TRet> getInitialValue,
```

Comment:
All overloads have a non-optional `Func` parameter to avoid the runtime resolution error that would happen otherwise.